src/libstd/path.rs

```diff
@@ -1035,6 +1035,49 @@ impl<'a> cmp::Ord for Components<'a> {
     }
 }
 
+/// An iterator over [`Path`] and its ancestors.
+///
+/// This `struct` is created by the [`parents`] method on [`Path`]. See its documentation for more.
+///
+/// # Examples
+///
+/// ```
+/// #![feature(path_parents)]
+///
+/// use std::path::Path;
+///
+/// let path = Path::new("/foo/bar");
+///
+/// for parent in path.parents() {
+///     println!("{}", parent.display());
+/// }
+/// ```
+///
+/// [`parents`]: struct.Path.html#method.parents
+/// [`Path`]: struct.Path.html
+#[derive(Copy, Clone, Debug)]
+#[unstable(feature = "path_parents", issue = "0")]
```

Comment:
This line raises two open questions for me:
1. Is the feature name appropriate?
2. What issue number can I use?

Reply:
The feature name seems OK to me, as long as it is distinct from any other feature name. :-)

I believe that if we decide to merge this, you need to create a tracking issue for stabilizing this feature, and that's the number that goes here.

I also think there is some docs where features are documented. But I don't know where it is.